Spec0/emojimap.json

```diff
@@ -0,0 +1,262 @@
+{
+  "version": "draft+3",
```

Comment:
I didn't see the first time that you had completed the list as well!
How come you put it in a different directory?